app/.gitignore

```diff
@@ -7,4 +7,5 @@
 /electron/i18n-helper.js
 /electron/preload.js
 /electron/windowSize.js
-electron/windowSize.test.js
\ No newline at end of file
+electron/windowSize.test.js
+plugin-management.js
```

Comment:
What about plugin-management-utils.js?

Needs a line at the end to keep github happy.

Reply:
`plugin-management-utils.js` isn't copied to app so not required here. Ill fix the EOL.